src/main/java/org/sagebionetworks/web/client/view/SettingsViewImpl.java

```diff
@@ -269,11 +269,7 @@ public SettingsViewImpl(
       presenter.goTo(new Quiz("Certification"))
     );
     certificationPassedButton.addClickHandler(event ->
-      DisplayUtils.newWindow(
-        "https://help.synapse.org/docs/User-Types.2007072795.html#UserAccountTiers-CertifiedUsers",
-        "_blank",
-        ""
-      )
+      presenter.goTo(new Quiz("Certification"))
```

Comment:
An unrelated fix for an issue that Xa noticed.  The "Certification Passed" button used to go to the documentation instead of showing the passing record.  Note that the documentation is still available from the More Info button below this one.